File: ClientApp/Config/Objects/Items/Car/Parts/Engine.ts
export default class Engine 
{
    public model: string;
    public number: string;
    public type: string;
    public fuel: string;
    public ecoClass: string;
    public cylindersCount: string;
    public cylindersVolume: string;
    public compression: string;
    public maxPower: string;
    public maxRotateMoment: string;
    public supplySystem: string;
    
    constructor(model: string, number: string, type: string, fuel: string, ecoClass: string, cylindersCount: string, 
                cylindersVolume: string, compression: string, maxPower: string, maxRotateMoment: string, supplySystem: string) 
    {
        this.model = model;
        this.number = number;
        this.type = type;
        this.fuel = fuel;
        this.ecoClass = ecoClass;
        this.cylindersCount = cylindersCount;
        this.cylindersVolume = cylindersVolume;
        this.compression = compression;
        this.maxPower = maxPower;
        this.maxRotateMoment = maxRotateMoment;
        this.supplySystem = supplySystem;
    }
}

File: ServerApp/Config/Objects/Items/Car/Parts/Engine.cs
using System;

namespace LBTDTools.ServerApp.Config.Objects.Items.Car.Parts
{
    public class Engine
    {
        public string Model { get; set; }
        public string Number { get; set; }
        public string Type { get; set; }
        public string Fuel { get; set; }
        public string EcoClass { get; set; }
        public string CylindersCount{ get; set; }
        public string CylindersVolume{ get; set; }
        public string Compression{ get; set; }
        public string MaxPower{ get; set; }
        public string MaxRotateMoment{ get; set; }
        public string SupplySystem{ get; set; }

        public string GetFullName() { return Model + " " + Number; }
    }
}